### PR TITLE
timaslj

### PR DESCRIPTION
**Pull request by timaslj**
Lab 2

### CODE REVIEW

File: Lab-2/timaslj_lab_2.ipynb [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CbPzq1tDwmaR",
+        "colab_type": "text"
+      },
+      "source": [
+        "### **Blocked matrix-matrix product**"
```
Comment (ajlace) [inline Lab-2/timaslj_lab_2.ipynb]: I am assuming this is an error.

File: Lab-2/timaslj_lab_2.ipynb [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+        "\n",
+        "      for i in range(size):\n",
+        "        for j in range(size):\n",
+        "          rand_int = random.randint(0,50)\n",
+        "          sym_A[i,j] = rand_int\n",
+        "          sym_A[j,i] = rand_int\n",
+        "      A, U = qr_algorithm(sym_A)\n",
+        "      \n",
+        "      id_matrix = np.identity(size)\n",
+        "      for i in range(size):\n",
```
Comment (ajlace) [inline Lab-2/timaslj_lab_2.ipynb:600]: Provide comments to clarify the tests.

**Review by ajlace — CHANGES_REQUESTED** [review]
I have requested 2 changes to lab 2. Other than those, it looks good.

File: Lab-2/timaslj_lab_2.ipynb (hunk 2 above)
Comment (timaslj) [reply Lab-2/timaslj_lab_2.ipynb:600]: Added!

File: Lab-2/timaslj_lab_2.ipynb (hunk 1 above)
Comment (timaslj) [reply Lab-2/timaslj_lab_2.ipynb]: Fixed! It should say QR eigenvalue algorithm.

**Review by ajlace — APPROVED** [review]
I approve of the changes, however it seems as the updated file is outside the Lab2-folder (it does not affect anything really)

**Pull request merged**